### PR TITLE
f-service@2.0.0-beta.4-service-updates-with-tests

### PR DESCRIPTION
**Pull request by smilburn1990**
### Added
- Created service package for shared logic
- Tests for the methods inside

### Removed
- Old imports for logic on root

### CODE REVIEW

File: packages/f-services/README.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+<div align="center">
```
Comment (pikax) [inline packages/f-services/README.md]: copy & paste from `f-header`, please improve this file.

File: packages/f-services/package.json [hunk 2]
```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@justeat/f-services",
+  "description": "Fozzie Services - Shared Services for Components and projects",
+  "version": "v2.0.0-beta.1",
```
Comment (pikax) [inline packages/f-services/package.json]: is it v2?

File: packages/f-services/package.json [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  "license": "Apache-2.0",
+  "engines": {
+    "node": ">=8.0.0"
+  },
+  "keywords": [
+    "fozzie"
+  ],
+  "scripts": {
+    "prepare": "yarn lint && yarn test && yarn build",
+    "build": "vue-cli-service build --target lib --name f-services ./src/index.js",
```
Comment (pikax) [inline packages/f-services/package.json]: is this a vue library?

**Review by simonsandersje — DISMISSED** [review]

File: packages/f-services/src/index.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
+/**
+ * @overview Fozzie services JavaScript behaviour.
+ *
+ * @module f-services
+ */
+
+
+// Import vue component
```
Comment (pikax) [inline packages/f-services/src/index.js:8]: not a vue component

File: yarn.lock [hunk 5]
```diff
@@ -1032,6 +1032,7 @@
 "@justeat/f-header@file:packages/f-header":
   version "2.0.0-beta.9"
   dependencies:
+    "@justeat/f-services" "file:../Library/Caches/Yarn/v4/npm-@justeat-f-header-2.0.0-beta.9-0468012f-b3f3-4f83-8f1b-8a3980760e6e-1569319556885/node_modules/@justeat/f-header/.yalc/@justeat/f-services"
```
Comment (pikax) [inline yarn.lock]: is this correct?

File: yarn.lock [hunk 6]
```diff
@@ -1062,6 +1063,12 @@
   dependencies:
     appboy-web-sdk "2.4.0"
 
+"@justeat/f-services@file:packages/f-header/.yalc/@justeat/f-services":
```
Comment (pikax) [inline yarn.lock]: local files...

**Review by pikax — CHANGES_REQUESTED** [review]

File: yarn.lock (hunk 6 above)
Comment (smilburn1990) [reply yarn.lock]: Need to publish the component first

File: packages/f-services/CHANGELOG.md [hunk 7]
```diff
@@ -0,0 +1,13 @@
+# Changelog
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+
+0.1.0
```
Comment (pikax) [inline packages/f-services/CHANGELOG.md:31]: the current version is: v2.0.0-beta.1

Comment (smilburn1990) [reply packages/f-services/CHANGELOG.md:31]: I believe this relates to the fozzie version? Might be wrong but that is what it is like in the meta component which was created recently.

File: packages/f-header/src/components/Navigation.vue [hunk 8]
```diff
@@ -286,7 +286,7 @@ export default {
                     this.userInfo = data;
                 }
             } catch (err) {
-                if (this.justLog) {
+                if (this.justLog.error) {
```
Comment (pikax) [inline packages/f-header/src/components/Navigation.vue]: just log has been declared as a function on the prop type, but you're using as an object.

File: packages/f-services/package.json [hunk 9]
```diff
@@ -0,0 +1,41 @@
+{
+  "name": "@justeat/f-services",
+  "description": "Fozzie Services - Shared Services for Components and projects",
+  "version": "v0.1.0",
```
Comment (pikax) [inline packages/f-services/package.json]: remove the `v`

**Review by pikax — CHANGES_REQUESTED** [review]

**Review by pikax — DISMISSED** [review]

**Review by simonsandersje — DISMISSED** [review]

File: packages/f-header/src/components/Header.vue [hunk 10]
```diff
@@ -63,7 +63,7 @@ export default {
         }
     },
     data () {
-        const locale = sharedService.getLocale(tenantConfigs, this.locale, this.$118n);
+        const locale = sharedServices.getLocale(tenantConfigs, this.locale, this.$118n);
         const localeConfig = tenantConfigs[locale];
         const theme = sharedService.getTheme(locale);
```
Comment (dandel10n) [inline packages/f-header/src/components/Header.vue]: Should be changed to `sharedServices` here too as on line 66

File: packages/f-services/README.md [hunk 11]
```diff
@@ -0,0 +1,30 @@
...
+
+    ```bash
+    yarn add @justeat/f-services
+    ```
+
+    ```bash
+    npm install @justeat/f-services
+    ```
+
+2.  Import the component
```
Comment (dandel10n) [inline packages/f-services/README.md]: Could we call it a package not a component maybe?

File: packages/f-services/src/index.js (hunk 4 above)
Comment (dandel10n) [reply packages/f-services/src/index.js:8]: +1

**Review by dandel10n — CHANGES_REQUESTED** [review]

**Review by dandel10n — APPROVED** [review]

**Review by simonsandersje — APPROVED** [review]

**Pull request merged**